solarforecastarbiter/validation/validator.py

```diff
@@ -380,3 +380,25 @@ def check_irradiance_day_night(times, solar_position=None, location=None,
     flags = pd.DataFrame(index=times, data=None, columns=['daytime'])
     flags['daytime'] = _check_limits(solar_position['zenith'], ub=max_zenith)
     return flags
+
+
+def check_timestamp_spacing(times):
+    """ Checks for even spacing of times.
+
+    Parameters
+    ----------
+    times : DatetimeIndex
+
+    Returns
+    -------
+    has_gaps : boolean
+        True
+    """
+
+    if times.size > 1:
+        dt = pd.Series(times.round('S').values)
```

Comment:
looks like you can use [pandas.unique](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.unique.html) directly on the times.

@alorenzo175 are we storing time stamps finer than 1 second? I think no.

Reply:
definitely won't store time stamps finer than 1 second

Reply:
Do you want to keep the rounding, or no?  I'm inclined to no, or make it a kwarg so we can round to 1 minute, for example.

Reply:
> looks like you can use pandas.unique directly on the times.

`pandas.diff()` operates on a series' values, which is why I'm making a series out of the index.  Open to better ideas.

Reply:
sorry, I mixed up a few things and what I said doesn't make any sense. 

I think you can use `unique` instead of `set`. Or `delta.unique()`. I'm guessing that it's significantly faster.